instagram.html

```diff
@@ -1,198 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
```

Comment:
A linguagem da escrita do texto do código, seguindo as boas práticas, deve ser em inglês. Porém, o site se for criado para uma empresa brasileira, por exemplo, deve manter sua lang="pt-br"